screens/Settings/About/index.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { render } from '@testing-library/react-native'
+import React from 'react'
+
+import About from './index'
+
+describe('About', () => {
```

Comment:
I know we have see this covered in cypress e2e tests. But I think it may be beneficial to have boilerplate integration test ready for this page.